cmd/pebble/cmd_signal.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2021 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/jessevdk/go-flags"
+
+	"github.com/canonical/pebble/client"
+)
+
+type cmdSignal struct {
+	clientMixin
+	Positional struct {
+		Signal   string   `positional-arg-name:"<SIGNAME>"`
+		Services []string `positional-arg-name:"<service>"`
+	} `positional-args:"yes" required:"yes"`
+}
+
+var shortSignalHelp = "Send a signal to one or more running services"
+var longSignalHelp = `
+The signal command sends a signal to one or more running services. The signal
+name must be uppercase to distinguish it from the service names, for example:
```

Comment:
Seems worth dropping the "to distinguish it from service names", as this is more of a UX detail, and it's not strictly true as we could just take the position into account. It just makes the CLI nicer and less ambiguous for a person to read and write.

Reply:
Done.